.github/workflows/nix.yaml

```
@@ -16,16 +21,19 @@ permissions:
+      github.event.action != 'closed'
     outputs:
       has-backend: ${{ steps.check-labels.outputs.has_backend }}
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout repository
+        uses: actions/checkout@v4
       
       - name: Apply labels
-        uses: actions/labeler@v4
         if: github.event_name == 'pull_request' && github.event.action != 'closed'
+        uses: actions/labeler@v4
```

actionlint (1.7.4)

the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)